apps/frontend/src/pages/projects/components/project-create-form/libs/constants/empty-string-length.constant.ts

```diff
@@ -0,0 +1,3 @@
+const EMPTY_STRING_LENGTH = 0;
```

Comment:
It looks like a common constant. Let's use more generic name and move it to the shared package 

```suggestion
const EMPTY_LENGTH_COUNT = 0;
```

Reply:
I'll call it `EMPTY_LENGTH`. Count sounds redundant